examples/Analytics.UpdateAction-result.json

```
@@ -0,0 +1,75 @@
+{
+  "@odata.context": "$metadata#SalesModel.SalesData/Sales(Customer(Country),Product(Name),Amount)",
```

Is this the correct way to address an entity set from a referenced schema?

---

I would expect

```suggestion
  "@odata.context": "$metadata#Sales(Customer(Country),Product(Name),Amount)",
```

because this response comes from the service that serves the `Sales` entity set.

Which means that the full example wouldn't work: if the response from `GET Sales?$apply=...` just mentions an action import name in the instance annotation, that action import must be in the same service / entity container as the entity set that is being aggregated.